docs/protocol/Staking.md

```diff
@@ -0,0 +1,25 @@
+# Staking
+
+Serai's staking pallet offers a DPoS system. All stake which enters the system
+is delegated somewhere. Delegatees can then allocate their stake to different
+validator sets, justifying their inclusion and providing financial security.
+
+Delegators may re-delegate stake whenever, so long as that stake isn't actively
+allocated. Delegators may also unstake whenever, so long as the prior condition
```

Comment:
While active/inactive works, I prefer "allocate" to "activate" since you *allocate* X to Y and A to B. Saying you *activated* 5 SRI to set 1 and 10 SRI to set 2 is... weird.

Reply:
Prefer "staked" and "bonded" myself. Seems simple to teach others the mechanism with those words.